Collection/src/Database.java

```diff
@@ -0,0 +1,81 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+class Database <T extends Print> implements Print {
```

Comment:
Print is a verb, you can't use verb for interface or class name, Printeable would sound better since the interface describes what class implementing it is **able to do**, Print_able_ .